core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -177,7 +179,7 @@ var ExplorationEditorPage = function() {
     const closeButton = element(by.css('.protractor-test-share-publish-close'));
     await waitFor.elementToBeClickable(
       closeButton, 'Close button taking too long to be clickable');
-    await closeButton.click();
+    await action.click('Close Button', closeButton);
     await waitFor.invisibilityOf(
       closeButton, 'Close button taking too long to disappear');
   };
```

Comment:
Below in `this.verifyExplorationSettingFields`, please make sure we wait for elements to load (e.g. be visible, present, clickable, etc.) before we interact with them

Reply:
This is the one throwing me the error right?